modules/plugin/coverage-multidim/netcdf/src/main/java/org/geotools/imageio/netcdf/utilities/NetCDFCRSUtilities.java

```diff
@@ -138,6 +146,9 @@
         VERTICAL_AXIS_NAMES.add("z");
         VERTICAL_AXIS_NAMES.add("depth");
         VERTICAL_AXIS_NAMES.add("pressure");
+
+        // Default is false, resulting into no automatic conversion anymore
+        CONVERT_AXIS_KM = Boolean.parseBoolean(System.getProperty(CONVERT_AXIS_KM_KEY, "false"));
```

Comment:
Note that this system property has been introduced to not break backward compatibility when backporting.
Backport will use "true" as default value, to have it working as it worked since today (doing the automatic conversion).
Only master has this set to false.

Reply:
You said in other parts of the code that the conversion could be due to using feet... so why make it specific to KM? Could be a CONVERT_AXIS_UNIT instead.

Reply:
I thought to name like that at the beginning.
However, there is no conversion at the moment for feet and we are going to stop having automatic-conversion... so an user setting that convert_axis_unit would expect to have it convert feet too which is actually not supported.
This is why it's explicitly named _km.

Reply:
Ok, did not see that there was already a special case for KM that you're trying to turn off. Go for it.